src/index.ts

```diff
@@ -47,6 +47,7 @@ export interface TypedMessage<T extends MessageTypes> {
     version?: string;
     chainId?: number;
     verifyingContract?: string;
+    salt?: ArrayBuffer;
```

Comment:
I chose `ArrayBuffer` for the expected type because it seemed like the most reasonable way to communicate raw bytes, since it's the underlying data structure for typed arrays. This is the only cross-platform way to represent bytes I think :thinking:.